### ISSUE TITLE
Harsh Joshi

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#171]

**Comment by aflies1** [comment]
Timing: 
Only taking 1 class (9-10:30) Tuesday / Thursday
Will need to be in the lab for a couple days a week
After 2pm will be free

Career Interest:
Past 4 years worked as a Data Engineer
Masters in data science
Internships as Data engineer and Data Scientist
Looking for something within that field
Manageing Data bases, projects, building some prediction models
Industry does not matter much, has financial industries experience in India

Expereience
In many different fields
Mainly analyst/ scientist

Skills
Python has worked 4+ years
Mongo DB, SQL
Hives
Predictive Modeling (a couple projects) - has done this from start to finish, planning, securing the data, and forward from there
ML? Has completed a ml class and currently working on some data mining classes

Why Masters?
Always wanted to learn more in ML take a deeper dive
MU is teaching what he wants to learn
Last semester was the basics, and now the high level things

Team?
In india has always been a team project, team of 10, manager would facilitate what needs to get done, goal is to complete the project as soon as possible. Woud work in teams of 3. Go get the data, then clean, then help with building the models
Jira / Asana


Will be located in the US through winter break

**Comment by aflies1** [comment]
Resume; https://drive.google.com/drive/folders/1nWwkVJCtvCkvgcx3xfonrgqEFwSh67dO

**Comment by ldaudet** [comment]
Harsh Joshi

He is a full CS background, working four years in india after his CS college degree in a python environment.

Accent is strong but very understandable, not an issue at all. Strange thing: he looks a LOT like my advisor at ND.

Lots of data analysis and manipulation experience while in India. He then switch to wanting to create some models for all that data.

Right now working in his master in data

He did two internships revolving on data, being data analyst and data engineer intern.

He also did a couple of projects in data science, one with the CORI fundation and one with ExcelR Private. All that experience was done in Python. 

Experienced in using Github as well.

In terms of ML, he is still exploring as it is his master he is currently doing. He has experience with XGBoost, and some experience in time series. For the time series, he did some basic time series. No experience in NLP.

Python is his main wheelhouse, but he also have experience in AWS ecosystem, and databases. His SQL foo is pretty strong for the experiences he had in the professional world before.

TL;DR: he has a lot of python experience and I think he would do great on the project with Prescient. Unfortunately, his ML background is too weak for the USAA or internal Algo team.
